packages/dev-frontend/src/components/Header.tsx

```diff
@@ -1,17 +1,17 @@
 import React from "react";
 import { Box, Container, Heading } from "theme-ui";
 import { Link } from "./Link";
-import { ThusdLogo } from "./ThusdLogo";
+import { GenericIcon } from "./GenericIcon";
 import { UserAccount } from "./UserAccount";
 
-const logoHeight = "38px";
+const logoHeight: string = "38px";
```

Comment:
I believe it's not needed to be added in theme.ts, since it's not used frequently as a variant across the app.
But added this info to be passed directly as prop instead of declaring a variable for it.